apps/extension/src/state/wallets.ts

```diff
@@ -35,6 +38,41 @@ export const createWalletsSlice =
         await local.set('wallets', [newWallet.toJson(), ...wallets]);
         return newWallet;
       },
+      getSeedPhrase: async () => {
+        // const passwordKey = get().password.key;
+        // const key = await Key.fromJson(passwordKey!);
+
+        // const activeWallet = getActiveWallet(get());
+
+        // if (!activeWallet) return [];
+
+        // const encryptedSeedPhrase = activeWallet.custody.encryptedSeedPhrase;
+        // const unsealeSeedPhrase = await key.unseal(encryptedSeedPhrase);
+
+        // return unsealeSeedPhrase?.split(' ') ?? [];
```

Comment:
@zpoken It looks like this was not cleaned up. We shouldn't commit commented out code. Please remove on next PR.